plugins/versionpress/tests/Unit/EntityInfoTest.php

```diff
@@ -129,6 +147,7 @@ public function commonEntityIsNotFalselyIdentifiedAsIgnored()
         $entity = [
             'some_field' => 'value',
             'other_field' => 'b',
+            'capitalized_value_field' => 'qqq',
```

Comment:
This test method (overall) seems to verify that more complex rules with multiple conditions, e.g., `'some_field:"value" other_field:"a"'`, match only when all their parts are satisfied. ("Common entity" in the test name is not helpful to me but I struggle coming up with a better suggestion 😄.)

The capitalized field example is a separate rule, `'capitalized_value_field: VALUE'`, so I don't think it belongs here.

Reply:
Makes sense to me - I've removed that line from the test.